mypyc/emit.py

```diff
@@ -170,18 +171,48 @@ def tuple_c_declaration(self, rtuple: RTuple) -> List[str]:
 
         return result
 
+    def tuple_undefined_check_cond(
+            self, rtuple: RTuple, tuple_expr_in_c: str,
+            c_type_compare_val: Callable[[RType], str], compare: str) -> str:
+        item_type = rtuple.types[0]
+        if not isinstance(item_type, RTuple):
+            return '{}.f0 {} {}'\
+                .format(tuple_expr_in_c, compare, c_type_compare_val(item_type))
```

Comment:
Put the `.format(` on the first line and break after the `(` instead of using a `\`